E-snippets.json

```diff
@@ -208,12 +208,12 @@
   " Get the name of the current file, editor, as a string.": {
     "prefix": "E.currentFileName ",
     "body": "E.currentFileName(${1:})$0",
-    "description": " Get the name of the current file, editor, as a string.\n *\n * For example, within my `~/init.js` pressing `cmd+e` on the following:\n * ```\n * E.currentFileName() //  ⇒  /Users/musa/init.js\n * ```\n "
```

Comment:
This just has `init.js` to `.init.js` renaming.

Okie dokies.

Reply:
Just make it portable.